docs/react-native.md

```diff
@@ -0,0 +1,232 @@
+# React Native Guide Use
+
+## Minimal Requirements
+
+- Kotlin: 1.8.22
+- Gradle: 7.6.1
+
+**Warning: This guide uses the older architecture of React Native, Android UI Components.**
+
+## Setup
+
+### Setting `android/build.gradle`
+
+```
+buildscript {
+    ext {
+        buildToolsVersion = "33.0.0"
+        minSdkVersion = 26
```

Comment:
```suggestion
```groovy
buildscript {
    ext {
        buildToolsVersion = "33.0.0"
        minSdkVersion = 26
```

Reply:
nit: adding the language after the first ``` will enable code highlighting